Dockerfile

```diff
@@ -156,7 +164,7 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 4F4EA0AAE5
 
 # install license_finder
 COPY . /LicenseFinder
-RUN bash -lc "cd /LicenseFinder && bundle install -j4 && rake install"
+RUN bash -lc "cd /LicenseFinder && bundle config set no-cache 'true' && bundle install -j4 && rake install"
```

Comment:
Sorry for another cache comment lol but will `bundle config --no-cache` work instead of `set no-cache 'true'`?

Reply:
@bluefish6 Bump :)